textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/DoubleField.java

```diff
@@ -0,0 +1,47 @@
+package edu.uci.ics.textdb.common.field;
+
+import edu.uci.ics.textdb.api.common.IField;
+
+public class DoubleField implements IField{
+    
+    private Double value;

```

Comment:
IField interface has Object getValue() method which needs to be implemented.. So using the wrapper class Double instead of primitive type double